wordvecspace/server.py

```diff
@@ -203,19 +208,28 @@ def get_nearest(
         v_w_i: Union[str, int, list, tuple],
         k: int = 512,
         metric: str = "angular",
+        distances: bool = False,
         combination: bool = False,
+        norm: bool = True,
     ) -> list:
         """
         get_nearest("india", 20) => [509, 3389, 486, 523, 7125, 16619, 4491, 12191, 6866, 8776, 15232, 14208, 5998, 21916, 5226, 6322, 4343, 6212, 10172, 6186]
         get_nearest(["ram", "india"], 5, metric='euclidean') => [[3844, 16727, 15811, 42731, 41516], [509, 3389, 486, 523, 7125]]
         get_nearest(['india', 'bosnia'], 10, combination=True) => [523, 509, 486]
         """
         if self._type == "mem" or self._type == "disk":
-            neg = self.wv.get_nearest(v_w_i, k, metric=metric, combination=combination)
+            neg = self.wv.get_nearest(
+                v_w_i,
+                k,
+                metric=metric,
+                combination=combination,
+                distances=distances,
+                norm=norm,
```

Comment:
@RamanjaneyuluIdavalapati Please change this to normalization so that it will same throughout the code

Reply:
@arrangeesh Please note.

Reply:
Done